FrontEnd/src/components/profileList/ProfileCard.jsx

```diff
@@ -0,0 +1,145 @@
+import css from "./ProfileCard.module.css";
+import { Badge, Typography } from "antd";
+import { useState } from "react";
+import { StarOutlined, StarFilled } from "@ant-design/icons";
+import { PropTypes } from "prop-types";
+import axios from "axios";
+import { useSWRConfig } from "swr";
+const { Paragraph } = Typography;
+
+// FIXME: will be changed once serializer will be changed to return display_name
+const regions = [
+  {
+    key: "Lviv region",
+    value: "Львівська область",
+  },
+  {
+    key: "Kyiv region",
+    value: "Київська область",
+  },
+];
+
+export default function ProfileCard(props) {
+  const isAuthorized = props.isAuthorized;
+  const data = props.data;
+
+  const [isSaved, setIsSaved] = useState(data.is_saved);
+
+  const profile = {
+    id: data.id,
+    companyName: data.name,
+    activities: !data.activities.length ? null : data.activities,
+    region: regions.find((region) => region.key == data.region).value,
+    address: data.address,
+    categories: !data.categories.length ? null : data.categories,
+    isSaved: isSaved,
+    commonInfo: data.common_info,
+  };
```

Comment:
I would wrap it in useMemo hook

Reply:
done, thanks. very useful thing, I'll mark it!